fairlearn/experimental/enable_metric_frame_plotting.py

```diff
@@ -0,0 +1,17 @@
+"""Enables Metric Frame Plotting
+
+The API and results of these estimators might change without any deprecation
+cycle.
+
+Importing this file dynamically sets the
+:class:`~metrics._error_plotter.plot_metric_frame` as an attribute of the
+`MetricFrame` module::
+    >>> # explicitly require this experimental feature
+    >>> from fairlearn.experimental import enable_metric_frame_plotting # noqa
+    >>> # now you can import normally from metrics
+    >>> from fairlearn.metrics import MetricFrame
```

Comment:
Will this automatically log a warning that an experimental feature is in use? Not essential, but if there's an easy way to do it, it might be nice.